storage/plugins/bibframe-linked-data/BibframeLinkedDataPlugin.php

```
@@ -0,0 +1,795 @@
+    private function buildGraph(array $book): array
+    {
+        $bookId  = (int) $book['id'];
+        $authors = $this->fetchAuthors($bookId);
+        $pub     = !empty($book['editore_id']) ? $this->fetchPublisher((int) $book['editore_id']) : null;
+        $genre   = !empty($book['genere_id'])  ? $this->fetchGenre((int) $book['genere_id'])     : null;
+
+        $title    = (string) ($book['titolo'] ?? '');
+        $subtitle = (string) ($book['sottotitolo'] ?? '');
+        $year     = (string) ($book['anno_pubblicazione'] ?? '');
+        $langCode = $this->langCode((string) ($book['lingua'] ?? ''));
+        $viafId   = (string) ($book['viaf_id'] ?? '');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$viafId'. (undefined)

(UnusedLocalVariable)